src/Jira/Api/Client/CurlClient.php

```diff
@@ -71,6 +71,9 @@ public function sendRequest($method, $url, $data = array(), $endpoint, Authentic
         curl_setopt($curl, CURLOPT_SSL_VERIFYHOST, 0);
         curl_setopt($curl, CURLOPT_VERBOSE, $debug);
         if ($isFile) {
+            if (defined('CURLOPT_SAFE_UPLOAD')) {
+                curl_setopt($curl, CURLOPT_SAFE_UPLOAD, false);
```

Comment:
Ah, I see now. The `createAttachment` method uses `@` for file upload according to https://docs.atlassian.com/jira/REST/latest/#api/2/issue/{issueIdOrKey}/attachments-addAttachment .

Then it's OK for now, but might be bad in long term from security point of view when an attacker has control over name of the filename, that is uploaded it can trick CURL into uploading any other expect specified file.